frontend/src/Board.jsx

```diff
@@ -0,0 +1,66 @@
+import "./Board.css";
+import { useState, useCallback, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import BoardList from "./components/board/BoardList";
+import CreatePostModal from "./components/board/CreatePostModal";
+
+import { useNavigate } from "react-router-dom";
+
+const INIT_RELOAD = 0;
+
+const Board = ({ currentBoardName }) => {
+  const [visibleCards, setVisibleCards] = useState([]);
+  const [reload, setReload] = useState(INIT_RELOAD);
+
+  const { boardId } = useParams();
+
+  useEffect(() => {
+    const getCards = async () => {
+      const response = await fetch(
+        `http://localhost:3000/boards/${boardId}/posts`
+      );
+      const data = await response.json();
+      setVisibleCards(data);
+      console.log(data);
+    };
+    getCards();
+  }, [reload]);
+
+  const navigate = useNavigate();
+
+  const [isCreatePostVisible, setCreatePostVisibility] = useState(false);
+
+  return (
+    <div className="Board">
+      <button
+        className="arrow"
+        onClick={() => {
+          navigate("/home");
+        }}
+      >
+        ←
+      </button>
+      <header>
+        <h1>Kudos Board</h1>
+        <h2> {currentBoardName ? currentBoardName : "Title"} </h2>
```

Comment:
`{currentBoardName && <h2> {currentBoardName} </h2>`} to hide <h2> when there is no title